### PR TITLE
add pytorch app tests

### PR DESCRIPTION
**Pull request by GavinGu07**

### CODE REVIEW

File: apps/run-app-tests-pytorch-pip.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    pip uninstall -y analytics-zoo
+    pip uninstall -y bigdl
+    pip uninstall -y pyspark
+}
+
+chmod +x ${ANALYTICS_ZOO_HOME}/apps/ipynb2py.sh
+
+set -e
+
+RUN_PART1=0
```
Comment (qiuxin2012) [inline apps/run-app-tests-pytorch-pip.sh:16]: we don't need so many parts here.

Comment (GavinGu07) [reply apps/run-app-tests-pytorch-pip.sh:16]: Get. I will update it.

File: apps/run-app-tests-pytorch-pip.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    pip uninstall -y analytics-zoo
+    pip uninstall -y bigdl
+    pip uninstall -y pyspark
+}
+
+chmod +x ${ANALYTICS_ZOO_HOME}/apps/ipynb2py.sh
+
+set -e
+
+echo "#15 start app test for pytorch face-generation"
```
Comment (qiuxin2012) [inline apps/run-app-tests-pytorch-pip.sh:16]: `#1`

Comment (GavinGu07) [reply apps/run-app-tests-pytorch-pip.sh]: I have deleted the number since there is only one test.

**Pull request merged**